### PR TITLE
Small UX hints

### PR DESCRIPTION
**Pull request by kennwhite**
Some small but hopefully useful UI improvements to help with new user setups.

### CODE REVIEW

**Comment by dguido** [comment]
Looks good! I made one comment regarding the SSH key path. What do you think?

**Comment by kennwhite** [comment]
👍

File: config.cfg [hunk 1]
```diff
@@ -74,9 +74,10 @@ congrats: |
   "#    Config files and certificates are in the ./configs/ directory.    #"
   "#              Go to https://whoer.net/ after connecting               #"
   "#        and ensure that all your traffic passes through the VPN.      #"
-  "#          Local DNS resolver and Proxy IP address: {{ local_service_ip }}     "
-  "#                     The p12 password is {{ easyrsa_p12_export_password }}      "
-  "#                  The CA key password is {{ easyrsa_CA_password }}      "
+  "#          Local DNS resolver and Proxy IP address: {{ local_service_ip }}         #"
+  "#                     The p12 password is {{ easyrsa_p12_export_password }}                     #"
+  "#                  The CA key password is {{ easyrsa_CA_password }}                 #"
+  "#            Shell access: ssh -i algo.pem root@{{ ansible_ssh_host }}          #"
```
Comment (kennwhite) [inline config.cfg]: I was assuming the reader had already cd'd to the configs directory, but super minor either way.

Comment (dguido) [inline config.cfg]: I'll just merge it as is and see how many issues get filed :-P

**Comment by gunph1ld** [comment]
The key name should not be hardcoded.
See the [config](https://github.com/trailofbits/algo/blob/master/config.cfg#L83-L86)

**Pull request merged**